composer.json

```
@@ -6,8 +6,9 @@
   "require": {
     "php": ">=5.6",
     "composer/installers": "^v1.12.0 || ^2.2",
-    "dealerdirect/phpcodesniffer-composer-installer": "^1.0.0",
-    "wp-coding-standards/wpcs": "dev-develop#fca9d9e"
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7 || ^1.0",
+    "wp-coding-standards/wpcs": "^2.3 | dev-develop#fca9d9e",
+    "automattic/vipwpcs": "^2.0.0"
```

```suggestion
     "dealerdirect/phpcodesniffer-composer-installer": "^1.0.0",
    "wp-coding-standards/wpcs": "dev-develop#fca9d9e as 2.3.0",
    "automattic/vipwpcs": "^2.0.0"
```

Using some functionality called [aliases](https://getcomposer.org/doc/articles/aliases.md) is a better way around this.